### DISCUSSION TITLE
http://www.gunfus.com/Adding_comments_to_blog/

### DISCUSSION THREAD

**Discussion by giscus** [arvera/blog#12] [Comments]
# http://www.gunfus.com/Adding_comments_to_blog/

I a few years ago I tried to implement comments for the blog but that time there was only a few ways to do it with jekyll minimal mistakes type of blog, and I never got it working. You needed a servless app and I just got busy with other things, move forward a couple of years and features and github now supports Apps and Discussion threads (giscus.app)[https://giscus.app/]

http://www.gunfus.com/Adding_comments_to_blog/

**Comment by arvera** [comment]
Testing the comments!
